schemas.sql

```diff
@@ -0,0 +1,112 @@
+
+-- keep track of all users (id, tag, name)
+
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    user_id VARCHAR NOT NULL,
+    user_tag VARCHAR NOT NULL,
+    user_name VARCHAR NOT NULL,
+    deleted_at TIMESTAMP
+);
+
+-- From rememberMessage.js
+-- 'content' is a VARCHAR and not TEXT because 
+-- discord has a limit of 2000 chars per message (4000 for nitro users)
+-- and VARCHAR has a limit of 10485760
+CREATE TABLE messages (
+    id SERIAL PRIMARY KEY,
+    user_id VARCHAR NOT NULL,
+    content VARCHAR NOT NULL,
+    message_id VARCHAR NOT NULL,
+    message_ts VARCHAR NOT NULL,
+    deleted_at TIMESTAMP
+);
```

Comment:
If this is representing messages sent in the discord server, we may want to include things like discord_server_id and channel_id etc, anything required to uniquely identify a message.